README.md

```diff
@@ -35,6 +35,40 @@ make
 
 This builds libmongocrypt.dylib and test-libmongocrypt, in the cmake-build directory. Note, the `CMAKE_PREFIX_PATH` must include the path to the BSON library installation directory if it was not the defaults.  Also note that if your project will also dynamically link to the BSON library, you will need to add `-DENABLE_SHARED_BSON=ON` to the `cmake` command line.
 
+## Installing libmongocrypt on macOS ##
+First install [Homebrew according to its own instructions](https://brew.sh/). Using Homebrew, install the following dependencies.
+```
+brew install mongo-c-driver cmake
```

Comment:
I had to run `brew link mongo-c-driver` as well but that may have been because I already had an older version of libmongoc installed.

Reply:
My rough understanding is that homebrew won't overwrite files it didn't create. So, if you had installed libmongoc before (either by copying files, or building/installing from source). Then homebrew requires the explicit `link` after install to overwrite those files.